apps/services/search-indexer/dev-services/README.md

```diff
@@ -1,51 +1,50 @@
 # Elasticsearch Search Dev Server
 
-### Dependencies
+## Dependencies
 
-You must have `docker` and `docker compose` installed and running on you machine.  
-**Docker must be given 4gb+ of ram**
+Ensure `docker` and `docker compose` are installed, with Docker allocated 4GB+ RAM.
 
-### Start the server
+## Start the Server
 
 ```bash
 yarn dev-services services-search-indexer
 ```
 
-In about 30 seconds
-You should have Elasticsearch running on `localhost:9200`  
-And Kibana running on port `localhost:5601`
+Wait ~30 seconds. Access Elasticsearch at `localhost:9200` and Kibana at `localhost:5601`.
 
-### Import data
+## Import Data
 
-**You must have access to Contentful for the indexer to work correctly**  
-To import map templates, create indice and import data run
+**Contentful access is required.**
 
-Requred env variables:
+Set environment variables:
 
-- `CONTENTFUL_HOST=preview.contentful.com` - Get from Contentful
-- `CONTENTFUL_ACCESS_TOKEN=` - Get from Contentful or by running `yarn get-secrets api`
+- `CONTENTFUL_HOST=preview.contentful.com`
+- `CONTENTFUL_ACCESS_TOKEN=` (obtain from Contentful or via `yarn get-secrets api`)
 - `ELASTIC_NODE=http://localhost:9200/`
 
+Create indices and import data:
+
 ```bash
 yarn nx run services-search-indexer:build
 yarn nx run services-search-indexer:migrate
 ```
 
-### Nice to know
+## Additional Information
 
-#### Dictionaries
+### Dictionaries
 
-Dictionaries for icelandic are pulled into the elastic docker container on build under `analyzers/`  
-The index template is not imported automagically so you must insert them manually...for now  
-There is no unseen functionality here, these dockerfiles are **not used in production**,  
-so if you add a plug to this docker file you must make sure it is also added to required deployments
+Icelandic dictionaries are in the Elasticsearch Docker container under `analyzers/`. Add necessary plugins to the Dockerfile for development.
 
-#### Local errors
+### Local Errors
 
-In case you see a `TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block` error in your local terminal, you can allow more disk usage with the following commands:
+If you encounter `TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block`, execute:
```

Comment:
Why not state that we are allowing more disk usage

Reply:
Why: It's redundant (can be gathered from context via the error message).

I counter: Why is the default allowed disk usage not higher, so that the developer never has to do this?